### PR TITLE
docs: fix typo in readme

### PR DESCRIPTION
**Pull request by CarleneCannon-Conner**
The Postgres adapter and watcher example in the README is broken:

Missing a comma after connectionString's value
const pgOptions = {
  connectionString: 'postgres://localhost',
  migrate: true
}
Both await newAdapter... and await newWatcher... will throw a SyntaxError: Unexpected identifier

### CODE REVIEW

**Review by simoneb — APPROVED** [review]

**Review by simoneb — COMMENTED** [review]
Can you explain why we need this changes, besides the error with the comma?

**Comment by CarleneCannon-Conner** [comment]
The [changes](https://github.com/nearform/fastify-casbin/pull/49/commits/da820d44049bbb9aeb673b526886f394e5269ed3) fix the 'Postgres adapter and watcher' example which used to throw a SyntaxError: Unexpected identifier error. This change is part of the original [issue](https://github.com/nearform/fastify-casbin/issues/46) 
> Both await newAdapter... and await newWatcher... will throw a SyntaxError: Unexpected identifier

As for the update to the 'Using programmatically assembled model', it is just some minor points like missing the import of plugin and the naming of app/fastify to to avoid undefined errors

File: README.md [hunk 1]
```diff
@@ -54,17 +54,29 @@ const fastify = require('fastify')()
...
+      watcher: await newWatcher(pgOptions) // the watcher
+    })
+  } catch (err) {
+    fastify.log.error(err)
+    process.exit(1)
+  }
+}
+registerCasbin().then(() => {
+  // Run the server after async registration
 })
```
Comment (simoneb) [inline README.md:71]: please don't change this, you're introducing an inconsistency with the previous example, and in generate this is not the recommended pattern to register plugins in fastify.

File: README.md [hunk 2]
```diff
@@ -85,6 +97,7 @@ fastify.get('/protected', async () => {
 import fastify from 'fastify'
 import { join } from 'path'
 import { Model, FileAdapter } from 'casbin'
+import plugin from 'fastify-casbin'
```
Comment (simoneb) [inline README.md:93]: let's call this `fastifyCasbin` instead, plugin is very generic

**Review by simoneb — APPROVED** [review]

**Pull request merged**